docs/UserGuide.md

```diff
@@ -252,33 +252,92 @@ Examples:
 
 - `delete-video /module CS2040 /lecture 1 /video 3` deletes the 3rd video in the results of the `list /module CS2040 /lecture 1` command
 
+### Tag a module
+
+Tag a specified module from the current list of modules in Le Tracker with a description
+
+Format: `tag {module_code} /tags {tag_1}, [{tag_2}, {tag_3}, ...]`
+
+- Tag a module of the specified `module_code` from the current list of modules in Le Tracker
+- `module_code` must belong to an existing module
+- `tag_1, tag_2, ...` must be of correct format
+
+Example:
+- `tag EG2310 /tags fun, hard` tags the module `EG2310` with the tags `fun` and `hard`
+
 ### Tag a lecture
 
-Tags a specified lecture from a specified module with a description
+Tag a specified lecture from a specified module with a description
 
-Format: `tag /module {module_code} /lecture {lecture_id} /description {tag_description}`
+Format: `tag {lecture_name} /mod {module_code} /tags {tag_1}, [{tag_2}, {tag_3}, ...]`
```

Comment:
`/mod` should be an optional field as the context sensitive system will auto inject the argument if the context is selected. Refer to `add` command for example

Multi-value optional fields should be of this format. `/tags {tag_1}[, {tag_2}[, {tag_3}, ...]]]` (same for tagging lecture and video as well as untagging)